src/main/java/ChatReques5.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
```

Comment:
Redundant annotation. The @Data annotation already includes @Setter, @Getter, @ToString, and @Builder, making these annotations unnecessary.